research-hub-web/package.json

```diff
@@ -106,7 +106,8 @@
     "tslint": "~6.1.0",
     "typescript": "~4.0.5",
     "wait-on": "^5.2.0",
-    "webdriver-manager": "^12.1.7"
+    "webdriver-manager": "^12.1.7",
+    "ngx-contentful-rich-text": "^0.4.0"
```

Comment:
should ngx-contentful-rich-text be under dependencies rather than devDependencies?